core/client/fs/src/main/java/alluxio/client/block/policy/LocalFirstAvoidEvictionPolicy.java

```diff
@@ -27,11 +27,16 @@
 
 import javax.annotation.concurrent.ThreadSafe;
 /**
- * A policy that returns local host first, and if the local worker doesn't have enough availability,
- * it randomly picks a worker from the active workers list for each block write.
- * If no worker meets the demands, return local host.
- * USER_FILE_WRITE_AVOID_EVICTION_POLICY_RESERVED_BYTES is used to reserve some space of the worker
- * to store the block, for the values mCapacityBytes minus mUsedBytes is not the available bytes.
+ * A policy that returns the local worker first, and if the local worker doesn't
+ * exist or have enough availability, will select the nearest worker from the active
+ * workers list with sufficient availability.
```

Comment:
Oh nice catch! Maybe you want to further explain "nearest" here. The definition of near is based on tier identity (calling `BlockLocationUtils.nearest()` -> `TiereddentityUtils.nearest)`